client/styles/abstracts/_variables.scss

```diff
@@ -1,5 +1,8 @@
 $base-font-size: 12;
 
+//break points
+$break-small: 600px;
```

Comment:
could you use the Bootstrap breakpoint numbers? thanks!

Reply:
Sure thing. Sorry about that.

Reply:
no worries!